### PR TITLE
https://github.com/lazy-suzy/lazysuzy-code/issues/184 : Point 4: inprogress, point 8,9 pending

### PR DESCRIPTION
**Pull request by EatPrayCode**

### CODE REVIEW

File: index.html [hunk 1]
```diff
@@ -265,7 +268,7 @@ <h4 class="title">Favorites</h4>
       </div>
       <div class="main-content container-fluid">
         <div class="row">
-          <div class="col-md-12">
+          <div class="col-md-12 tool-bar-holder" id="tool-bar-holder">
```
Comment (codefromhimanshu) [inline index.html:271]: @EatPrayCode Why are we using both id and class here?

File: styles.css [hunk 2]
```diff
@@ -1822,3 +1832,180 @@ i.background-msg-arrow {
...
+
+/* Auth Related, Needs refactoring */
+
+.modal-dialog {
+  width: 35%;
+}
+
+.modal-heading {
+  text-align: center;
+  margin-top: 15%;
```
Comment (codefromhimanshu) [inline styles.css:1845]: Change the values of margin and padding in pixels

**Review by codefromhimanshu — CHANGES_REQUESTED** [review]

File: index.html (hunk 1 above)
Comment (EatPrayCode) [reply index.html:271]: There is some necessary to get by id, incases like hiding and showing the toolbar.
yes we can get by class, but then we need to do getByClass[0] which is little bit cumbersome.
Can we do this change once complete toolbox hiding functionality is done.

File: styles.css (hunk 2 above)
Comment (EatPrayCode) [reply styles.css:1845]: Ok

**Pull request merged**